.github/workflows/build-test-prValidation.yml

```diff
@@ -101,6 +101,53 @@ jobs:
           -Dpekko.log.timestamps=true \
           validatePullRequest
 
+  pekko-classic-remoting-tests:
+    name: Pekko Classic Remoting Tests
+    runs-on: ubuntu-22.04
+    if: github.repository == 'apache/pekko'
+    strategy:
+      fail-fast: false
+      matrix:
+        command:
+          - cluster/test distributed-data/test cluster-tools/test cluster-metrics/test
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
needs branch

Reply:
when the PR targets a branch, GitHub CI seems intelligent enough to checkout the PR branch

Have a look at https://github.com/apache/pekko/blob/1.0.x/.github/workflows/build-test-prValidation.yml -- no refs for any checkouts.

The nightly runs are different because we have the ymls in the main branch so if we want to test a branch, we need to adjust the checkouts in that case.